examples/test_performance/change_nfs.py

```diff
@@ -0,0 +1,33 @@
+###This script modifies the example_nf_deploy.json file so that it adds an additional simple_forward nf
```

Comment:
instead of just adding one more NF to the json file, I think it is better to have this program take a command line argument for the number of NFs to put in a chain.

It will also be more resilient if it does not directly modify the `example_nf_deploy.json` file. Instead, it should treat that file as a template that it loads, but then it builds its own file (possibly specified by another command line argument).  This way you don't need to assume the file starts with a particular state and it will also allow us to easily create chains of arbitrary lengths (maybe we only care about 2, 5, 16, 45, and 64 NF chains for some reason).

Reply:
Ok, sounds hard but doable